README.md

```diff
@@ -7,6 +7,8 @@ These statements can be dispatched to an LRS with xapiwrapper.min.js using a cus
 
 Check out http://adlnet.github.io/xapi-youtube for a live demo.
 
+This script also uses [xAPI Launch](https://github.com/adlnet/xapi-launch) to establish the connection to the LRS from our application.
+
```

Comment:
Add something here explaining launch allows you to send a grouping URL and that we've been using launch to group our demos into certain workshops. Also explain that launch is optional and you are free to set the context in the else block to whatever you want (and can even update the context in the .js file if need be).